src/dev/build/tasks/build_packages_task.ts

```diff
@@ -53,9 +54,9 @@ export const BuildXpack: Task = {
     });
 
     log.info('copying built x-pack into build dir');
-    await scanCopy({
-      source: config.resolveFromRepo('x-pack/build/plugin/kibana/x-pack'),
-      destination: build.resolvePath('x-pack'),
+    await cpy('**/{.,}*', build.resolvePath('x-pack'), {
+      cwd: config.resolveFromRepo('x-pack/build/plugin/kibana/x-pack'),
+      parents: true,
     });
```

Comment:
What was wrong with the previous version? `scanCopy()` is pretty efficient and doesn't require weird globs, it just copied a directory from one place to another.

Reply:
The previous version of `scanCopy` was replacing copy replacing the entire folder while that one is just copying its contents. As the task that builds and copies the Bazel packages happens before that I had to do that. I thought about just inverting the order of the tasks as building and copy Bazel packages is a by package action, however the x-pack build tasks depends on `@kbn` packages and in order to make sure those are up to date and build before using them we need to run the task before.